packages/lit-element/rollup.bundle.config.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @license
+ * Copyright (c) 2018 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {litBundle} from '../../rollup-common.js';
+
+export default litBundle({
```

Comment:
You can export an array of configs in a rollup config -- maybe it would be simpler to just have the one `lit-element/rollup.config.js` that looks like this?

```js
export default [
  ...litProdConfig(...),
  ...litMonoBundleConfig(...),
]
```

Reply:
I decided to keep it separate to make sure it didn't get published to npm accidentally.